### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by ebell495**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ ansi_term = "~0.12.1"
 color-eyre = "~0.6.1"
```
Comment (xansec) [inline Cargo.toml]: Looks like this just references the existing sashimi lib rather than hardcoding a version.

**Review by xansec — CHANGES_REQUESTED** [review]
I'm not sure about the modified top level Cargo.toml. Is that necessary? Please make the requested changes.

File: Cargo.toml (hunk 1 above)
Comment (ebell495) [reply Cargo.toml]: Updated

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**